profiler/src/encrypt.py

```diff
@@ -0,0 +1,18 @@
+from cryptography.fernet import Fernet
+
+from src.config import Config
+
+
+FERNET = Fernet(Config.secret)
+
+
+def encrypt(message: str) -> str:
+    return FERNET.encrypt(message.encode()).decode('utf-8')
```

Comment:
What about to use that thing that we talking about?

Hashing without opportunity to unhash password